### ISSUE TITLE
Submitted to Auto-Approve

### ISSUE THREAD

**Issue by okaycj** [lookit/lookit-api#999]
# TL;DR
Classically, Lookit has always reverted to the state of a study to `rejected` when any aspect of a study is changed.  Moving forward, we'd like to be able to conditionally move the state to `autoaccept` when non-essential values are changed. 

# Narrative
Currently, Lookit doesn't have a pattern to handle conditional state changes.  Thankfully, the library we're using does have [this functionality available](https://github.com/pytransitions/transitions#conditional-transitions).  We should update this library to the current stable version and implement a solution to handle a conditional change on a few fields.  _(These fields have yet to be determined.)_

Additionally, this state change will need to be viewable in the Study Details Log.  

# Acceptance Criteria
A precise description of how/when this issue will be considered closed, e.g.:

- [ ] Tests for conditional state change.
- [ ] UI Show autoaccept in logs

# Implementation Notes
The model will need a method to facilitate the conditional state change.  This is covered in the `transitions` docs linked above.

**Comment by mekline** [comment]
Here are some additional proposed acceptance criteria: 

[ ] The following list of study fields should be exempt from requiring admin review:

criteria,
max_age_months,
max_age_years,
min_age_months,
min_age_years,
max_age_days,
min_age_days,
criteria_expression

[ ] Text on the warning box that pops up when a user tries to save edits to a currently running study should mention that some minor changes will get automatically re-approved. Right now it's a warning of some kind; mks is not certain what the exact current text of that popup is.

**Comment by mekline** [comment]
Also proposal: Rename "APPROVED" to "ADMIN-APPROVED" and add "AUTO-APPROVED" as a new state?
